src/pages/storage/forms/UploadVolumeBackupFileForm.tsx

```diff
@@ -0,0 +1,284 @@
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  useNotify,
+} from "@canonical/react-components";
+import { useCurrentProject } from "context/useCurrentProject";
+import StoragePoolSelector from "pages/storage/StoragePoolSelector";
+import type { ChangeEvent, FC } from "react";
+import { useCallback, useState } from "react";
+import { fileToSanitisedName } from "util/helpers";
+import type { LxdStorageVolume, UploadState } from "types/storage";
+import { useEventQueue } from "context/eventQueue";
+import { useToastNotification } from "context/toastNotificationProvider";
+import { useQueryClient } from "@tanstack/react-query";
+import { useNavigate } from "react-router-dom";
+import { queryKeys } from "util/queryKeys";
+import { useFormik } from "formik";
+import type { AxiosError } from "axios";
+import type { LxdSyncResponse } from "types/apiResponse";
+import * as Yup from "yup";
+import classnames from "classnames";
+import ResourceLink from "components/ResourceLink";
+import ResourceLabel from "components/ResourceLabel";
+import { uploadVolume } from "api/storage-volumes";
+import {
+  linkForVolumeDetail,
+  testDuplicateStorageVolumeName,
+} from "util/storageVolume";
+import ClusterMemberSelector from "pages/cluster/ClusterMemberSelector";
+import { useStoragePools } from "context/useStoragePools";
+import { isClusterLocalDriver } from "util/storagePool";
+import { useClusterMembers } from "context/useClusterMembers";
+
+export interface UploadVolumeBackupFileFormValues {
+  volumeFile: File | null;
+  name: string;
+  pool: string;
+  targetClusterMember: string;
```

Comment:
```suggestion
  clusterMember: string;
```
Let's go with a standard name.

The current custom name breaks the check in `testDuplicateStorageVolumeName`. Because on a cluster, an import with the name that already exists on another cluster member is wrongfully detected as a duplicate. I.e. when importing to another cluster member where a volume with that name does not exist yet.